csrankings-p.csv

```diff
@@ -140,6 +140,7 @@ Parampalli Udaya,University of Melbourne,http://people.eng.unimelb.edu.au/udaya,
 Parasara Sridhar Duggirala,University of North Carolina,http://cs.unc.edu/~psd,00S6A4IAAAAJ
 Paraschos Koutris,University of Wisconsin - Madison,http://pages.cs.wisc.edu/~paris,CaO1wNAAAAAJ
 Pari Delir Haghighi,Monash University,http://monash.edu/research/explore/en/persons/pari-delir-haghighi(70b2eef2-f84e-44ac-b48b-381f2ab4ec69).html,zvNKqskAAAAJ
+Parikshit Maini,University of Nevada,,9GEAUlcAAAAJ

```

Comment:
Missing web page.

Reply:
Bump @eelkef

Reply:
hi, due to Visa issues we had to postpone his hire date to 9.1.22. We can either remove him for or just add: https://www.unr.edu/cse/people/parikshit-maini which will be his website (it is already created in our CMS but not published yet).

Reply:
Home page still dead.

Reply:
Please fix; I will be closing old PRs this week.

Reply:
Hi Emery, sorry for following up late. Parikshit has not been able to join us yet due to visa issues. I deleted adding him to our department.

Reply:
Unfortunately, you deleted csrankings-p.csv from the PR, which would delete it from the repository if it's merged. Please fix or open a new PR.